mem0/configs/vector_stores/chroma.py

```diff
@@ -1,15 +1,26 @@
+import subprocess
+import sys
 from typing import Optional, ClassVar, Dict, Any
 
 from pydantic import BaseModel, Field, model_validator
 
+try:
+    from chromadb.api.client import Client
+except ImportError:
+    user_input = input("The 'chromadb' library is required. Install it now? [y/N]: ")
+    if user_input.lower() == 'y':
+        try:
+            subprocess.check_call([sys.executable, "-m", "pip", "install", "chromadb"])
+            from chromadb.api.client import Client
+        except subprocess.CalledProcessError:
+            print("Failed to install 'chromadb'. Please install it manually using 'pip install chromadb'.")
+            sys.exit(1)
+    else:
+        print("The required 'chromadb' library is not installed.")
+        sys.exit(1)
+
 
 class ChromaDbConfig(BaseModel):
-    try:
```

Comment:
Hey @Jai0401 Really sorry my bad, can you please keep the import statement inside the class.